Signup/Signup.css

```diff
@@ -0,0 +1,59 @@
+@import url(https://king-ui.netlify.app/styles.css);
+@import url(https://king-ui.netlify.app/Stylesheets/navbar.css);
+
+@import url(https://king-ui.netlify.app/Stylesheets/reset.css);
+
+body {
+    margin: 0;
+    display: block;
+}
+
+nav {
+    display: flex;
+    justify-content: space-around;
+    border-bottom: 2px solid var(--primary-color);
+    align-items: center;
+    background-color: white;
+}
+nav .text-input {
+    width: 50%;
+    padding: 0.5rem 1rem;
+}
```

Comment:
Most of this css is already added in your styles.css file. You don't need to rewrite it here instead you can just import that file

Reply:
i have imported, wanted some custom changes